x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -80,9 +82,34 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
     sourceIndex,
     sourceIndexNameEmpty,
     sourceIndexNameValid,
+    sourceIndexContainsNumericalFields,
+    sourceIndexFieldsCheckFailed,
     trainingPercent,
   } = form;
 
+  // Find out if index pattern contain numeric fields. Provides a hint in the form
+  // that an analytics jobs is not able to identify outliers if there are no numeric fields present.
+  const validateSourceIndexFields = async () => {
+    try {
+      const indexPattern: IndexPattern = await kibanaContext.indexPatterns.get(
+        indexPatternsMap[sourceIndex]
+      );
+      const containsNumericalFields: boolean = indexPattern.fields
+        .filter(f => !OMIT_FIELDS.includes(f.name))
+        .map(f => f.type)
+        .includes('number');
```

Comment:
no need to filter and map the whole collection, just fist match is enough 
```suggestion
       const containsNumericalFields: boolean = indexPattern.fields
        .some(({ name, type }) => !OMIT_FIELDS.includes(name) && type === 'number');
```

Reply:
Good one! 👍 Updated in 510e462da49149f9747e14e4216ee46587996cb0